<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting_camera</class>
 <widget class="QWidget" name="setting_camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera setting</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Brand:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotate:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Camera 0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera 5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera 6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera 7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera 8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera 9</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_id">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_conn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>GigE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USB 3</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_Brand">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Basler</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CSR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WC + DSHOW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WC + Default</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Point Grey</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IP Camera</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hik Vision</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Imaging source</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_Flip">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vertically</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Horizontally</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Both</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_Rotate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90ClockWise</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90CounterClockWise</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>180</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_thoat">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tho√°t</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
